features/admin.actions.v1/components/action-config-form.tsx

```diff
@@ -93,9 +97,10 @@ const ActionConfigForm: FunctionComponent<ActionConfigFormInterface> = ({
         (state: AppState) => state.config.ui.features.actions);
     const [ isAuthenticationUpdateFormState, setIsAuthenticationUpdateFormState ] = useState<boolean>(false);
     const [ authenticationType, setAuthenticationType ] = useState<AuthenticationType>(null);
-    const [ isSubmitting, setIsSubmitting ] = useState(false);
-    const [ isShowSecret1, setIsShowSecret1 ] = useState(false);
-    const [ isShowSecret2, setIsShowSecret2 ] = useState(false);
+    const [ isSubmitting, setIsSubmitting ] = useState<boolean>(false);
+    const [ isShowSecret1, setIsShowSecret1 ] = useState<boolean>(false);
+    const [ isShowSecret2, setIsShowSecret2 ] = useState<boolean>(false);
```

Comment:
Shall we use meaningful variable names for these state variables?

Reply:
Need to check and fix. I just added the missing types.